modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+// * storage/persistent_storage/persistent-storage-csi-aws-efs.adoc
+// * storage/container_storage_interface/persistent-storage-csi-aws-efs.adoc
+// * storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc
+// * storage/container_storage_interface/rosa-persistent-storage-aws-efs-csi.adoc
+
+:_content-type: PROCEDURE
+[id="efs-sts_{context}"]
+= Configuring an Operator to use Amazon Web Services Secure Token Service
+
+You can configure Operators on Amazon Web Services (AWS) clusters to use the AWS Secure Token Service (STS). Operators that use STS assign components IAM roles that provide short-term, limited-privilege security credentials.
+
+Perform this procedure after installing the an Operator that can use STS, but before installing any required drivers for the Operator. If you perform this procedure after installing a driver for an Operator, the Operator might not be able to function correctly.
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. You can use the `--dry-run` flag to avoid making API calls. Using this flag creates JSON files on the local file system instead. You can review and modify the JSON files and then apply them with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* You have an AWS cluster that is configured to use STS.
+
+* You have the {product-title} release image that was used to install the cluster.
+
+* You have extracted and prepared the `ccoctl` binary during cluster installation. For more information, see "Configuring the Cloud Credential Operator utility".
```

Comment:
These two feel a little weird to me. Basically, we are asking them to remember what they did during install and make sure they have everything still in the same state. I am not aware of similar prereqs in our content but perhaps there are.

Reply:
Yeah! I don't believe the user can do all of it if he already installed the cluster. To simplify those steps I would say that he can:
- extract the release image version from the cluster (CR `ClusterVersion`)
- extract the cloud-credentials-operator image from the release
- extract the `ccoctl`

```bash
RELEASE_IMAGE=$(oc get clusterversion version -o jsonpath='{.status.desired.image}')
CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator')
oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ${PULL_SECRET_FILE}
chmod 775 ccoctl
```

Reply:
ok, so basically the same prep as if we are starting from scratch on an install. Will grab that content :+1: